tests/integration/galley/webhook/webhook_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package galley
+
+import (
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+var (
+	vwcName    = "istio-galley"
+	deployName = "istio-galley"
+	sleepDelay = 3 * time.Second // How long to wait to give the reconcile loop an opportunity to act
```

Comment:
Nit: I'd increase this to a larger value (e.g. 10 or 15 seconds). In resource constrained environments things can take a long time to occur.

Reply:
Done